evalml/tests/component_tests/test_utils.py

```diff
@@ -236,6 +247,67 @@ def test_estimator_unable_to_handle_nans():
         estimator_unable_to_handle_nans("error")
 
 
+def test_test_downcast_nullable_types_can_handle_no_schema():
```

Comment:
These tests should live in the woodwork utils tests file, not the component utils file.